frontend/src/functions/chats.ts

```diff
@@ -0,0 +1,20 @@
+import { StreamChat, DefaultGenerics } from 'stream-chat';
+
+export function getChatClient(): StreamChat<DefaultGenerics> {
+  const apiKey = process.env.REACT_APP_STREAM_API_KEY
+    ? process.env.REACT_APP_STREAM_API_KEY
+    : '';
+
+  return StreamChat.getInstance(apiKey);
+}
+
+export function getChannel(
+  chatClient: StreamChat<DefaultGenerics>,
+  userID: string
```

Comment:
Just like we did for `getChatClient` let's add a return type for this function too